### DISCUSSION TITLE
Whether Strimzi Operator 0.18.0 supports kuberentus version 1.20 or not

### DISCUSSION THREAD

**Discussion by tianhao98** [strimzi/strimzi-kafka-operator#10822] [Q&A] [answered]
Strimzi Operator 0.18.0 is displayed on the [download](https://strimzi.io/downloads/#support-footnotes) page and supports Kubernetes 1.11. I want to know what the range is and whether Kubernetes 1.20 is supported

**Comment by scholzj** [comment]
Kube 1.20 was released only after Strimzi 0.18. It will probably work, but you will have to try it out.

**Reply by scholzj** [reply]
BTW: Both versions are super old and long _out of support_. You should not be using any of them.

**Reply by tianhao98** [reply]
Okay, I'm currently doing a Kubernetes upgrade, and I'm in the process of upgrading.

**Reply by scholzj** [reply, marked as answer]
Well, good luck. Keep in mind that Strimzi 0.18 definitely does not support Kube 1.22 or newer.

**Reply by xiaoxiongxyy** [reply]
I want to understand why K8s 1.22 must not support it? Is it because K8s has made changes to the underlying CRD?

**Reply by scholzj** [reply]
There were some major CRD changes on Kubernetes side at some point. But I do not remember anymore the exact versions of Kubernetes it affected and the Strimzi version it affected. But this will be recorded on the website and in the docs.
